packages/doenetml/src/Viewer/DocViewer.tsx

```diff
@@ -309,171 +314,38 @@ export function DocViewer({
-            } else if (e.data.messageType === "copyToClipboard") {
-                copyToClipboard(e.data.args);
-            } else if (e.data.messageType === "navigateToTarget") {
-                navigateToTarget(e.data.args);
-            } else if (e.data.messageType === "navigateToHash") {
-                navigate(location.search + e.data.args.hash, {
-                    replace: true,
-                });
-            } else if (e.data.messageType === "reportScoreAndState") {
-                if (reportScoreAndStateCallback) {
-                    reportScoreAndStateCallback({
-                        ...e.data,
-                        activityId,
-                        docId,
-                    });
-                } else {
-                    window.postMessage({
-                        ...e.data,
-                        subject: "SPLICE.reportScoreAndState",
-                        activityId,
-                        docId,
-                    });
-                }
-            } else if (e.data.messageType === "recordSolutionView") {
-                window.postMessage({
-                    ...e.data,
-                    subject: "SPLICE.recordSolutionView",
-                    activityId,
-                    docId,
-                });
-            } else if (e.data.messageType === "sendEvent") {
-                window.postMessage({
-                    ...e.data,
-                    subject: "SPLICE.sendEvent",
-                    activityId,
-                    docId,
-                });
-            } else if (e.data.messageType === "documentStructure") {
-                documentStructureCallback?.({
-                    ...e.data,
-                    activityId,
-                    docId,
-                });
-            } else if (e.data.messageType === "terminated") {
-                reinitializeCoreAndTerminateAnimations();
-            }
-        };
+        // coreWorker.current.onmessage = function (e) {
+        //     // console.log("message from core", e.data);
+        // if (e.data.messageType === "navigateToTarget") {
+        //         navigateToTarget(e.data.args);
+        //     } else if (e.data.messageType === "navigateToHash") {
+        //         navigate(location.search + e.data.args.hash, {
+        //             replace: true,
+        //         });
+        //     } else if (e.data.messageType === "recordSolutionView") {
+        //         window.postMessage({
+        //             ...e.data,
+        //             subject: "SPLICE.recordSolutionView",
+        //             activityId,
+        //             docId,
+        //         });
+        //     } else if (e.data.messageType === "sendEvent") {
+        //         window.postMessage({
+        //             ...e.data,
+        //             subject: "SPLICE.sendEvent",
+        //             activityId,
+        //             docId,
+        //         });
+        //     }
+        // };
```

Comment:
Why is this still here?

Reply:
I still need to add back this functionality (or for the "navigate" functions, decide if that is still part of the feature set).